### PR TITLE
RET-3675. RET-3397. Fix application properties

### PR DESCRIPTION
**Pull request by Itay-Kirsh**
https://tools.hmcts.net/jira/browse/RET-3675
Move defaults for template ids into application.properties and remove duplicates from values.yaml

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Comment by Itay-Kirsh** [comment]
Checked that all templates work and can be called. Doesn't fix all of 3675 but doing it in parts

**Review by brendanmchugh — APPROVED** [review]

File: config/owasp/suppressions.xml [hunk 1]
```diff
@@ -43,5 +43,11 @@
...
         <cve>CVE-2023-35116</cve>
     </suppress>
-
+    <suppress>
+        <notes><![CDATA[
+   file name: okio-jvm-2.8.0.jar
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/com\.squareup\.okio/okio@.*$</packageUrl>
+        <cpe>cpe:/a:squareup:okio</cpe>
+    </suppress>
```
Comment (JackReevies) [inline config/owasp/suppressions.xml:52]: Is there a reason for suppressing this?

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReferralHelper.java [hunk 2]
```diff
@@ -524,14 +524,15 @@ public static List<String> validateEmail(String email) {
...
      * @param referralNumber The number of the referral.
      * @param username Name of the user making or replying to this referral.
+     * @param linkToExui link to Exui with the caseId as a parameter
      * @param isNew Flag for if this is a new referral.
      */
-    public static Map<String, String> buildPersonalisation(CaseDetails detail, String referralNumber, boolean isNew,
-                                                           String username) {
-        CaseData caseData = detail.getCaseData();
+    public static Map<String, String> buildPersonalisation(CaseDetails details, String referralNumber, boolean isNew,
+                                                           String username, String linkToExui) {
```
Comment (JackReevies) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReferralHelper.java:534]: minor: do we need to pass in CaseDetails now? looks like passing CaseData would suffice

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SendNotificationService.java [hunk 3]
```diff
@@ -249,7 +248,7 @@ private void sendRespondentEmail(CaseData caseData, Map<String, String> emailDat
     private Map<String, String> buildPersonalisation(CaseDetails caseDetails, String envUrl) {
         return Map.of(
                 CASE_NUMBER, caseDetails.getCaseData().getEthosCaseReference(),
-            "environmentUrl", envUrl + caseDetails.getCaseId()
+            "environmentUrl", envUrl
         );
```
Comment (JackReevies) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SendNotificationService.java:252]: Single line now?

File: src/main/resources/application-dev.properties [hunk 4]
```diff
@@ -21,40 +21,6 @@ tornado.accessKey=${TORNADO_ACCESS_KEY:}
...
-
-nocNotification.template.respondent.id=a3539d79-65c0-491c-b578-b58cf321f83e
-nocNotification.template.claimant.id=3d0c5784-0055-4863-9c03-7d37d9b2ad8d
-nocNotification.template.previousrespondentsolicitor.id=fe52b39f-852c-43ca-a42a-b9a27c43b130
-nocNotification.template.newrespondentsolicitor.id=8fe52f24-40b2-4986-8f86-4dd1af311cbd
-nocNotification.template.tribunal.id=1d5efcbd-1971-4ebe-bfe8-72ba36b5abac
-
-sendNotification.template.id=1b8299d7-5051-4b78-a870-726b6384d8a8
-respondNotification.noResponseTemplate.id=d2caad87-2bab-4bc0-b598-01f1a7341f9e
-
```
Comment (JackReevies) [inline src/main/resources/application-dev.properties:57]: 👌

**Review by JackReevies — CHANGES_REQUESTED** [review]

File: config/owasp/suppressions.xml (hunk 1 above)
Comment (Itay-Kirsh) [reply config/owasp/suppressions.xml]: no, undone

**Review by JackReevies — APPROVED** [review]

**Review by brendanmchugh — APPROVED** [review]

**Pull request merged**